src/main/java/h13/model/gameplay/sprites/Enemy.java

```diff
@@ -56,7 +56,7 @@ public Enemy(final int xIndex, final int yIndex, final double velocity, final in
      *
      * @return The enemy's X-index of the enemy grid.
      */
-    public int getxIndex() {
+    public int getXIndex() {
```

Comment:
Auch wenn du von der Konvention her hier recht hast muss ich nochmal überlegen ob das mit wenig Aufwand in den Tests anpassbar wird beide Versionen zu akzeptieren... (wahrscheinlich muss ich was mit reflections basteln)

Reply:
Fixed by [bcfd578](https://github.com/FOP-2223/FOP-2223-H13-Student/pull/2/commits/bcfd578f5945df8f3066f67fbdd7234d756a105e)